package.json

```diff
@@ -27,8 +27,7 @@
   "author": "Adam Gruber <adamgruber@me.com>",
   "license": "MIT",
   "dependencies": {
-    "camel-case": "^3.0.0",
-    "jest-cli": "^23.6.0"
+    "camel-case": "^3.0.0"
```

Comment:
If you rebase then this fix commit isn't necessary and the package-lock file won't be updated.